quantum/split_common/transport.c

```diff
@@ -109,7 +116,10 @@ void transport_slave(matrix_row_t matrix[]) {
     sync_timer_update(i2c_buffer->sync_timer);
 #    endif
     // Copy matrix to I2C buffer
-    memcpy((void *)i2c_buffer->smatrix, (void *)matrix, sizeof(i2c_buffer->smatrix));
+    memcpy((void*)i2c_buffer->smatrix, (void *)slave_matrix, sizeof(i2c_buffer->smatrix));
```

Comment:
transport_slave function parameters have not been changed to support the slave matrix here.   This will fail to compile if you define `USE_I2C`

Reply:
bah, solid catch, stupid rebase and merge failures.

Reply:
fixed